Assets/Scripts/NPCController.cs

```diff
@@ -9,8 +9,11 @@ public class NPCController : MonoBehaviour
 
     public string npcName = "NPC";
 
+    public GameObject npcChatUI;
+
     private void Start()
     {
+        npcChatUI.SetActive(false);
```

Comment:
Might be worth considering doing this when the player walks away without clicking the x button, do it in a later story tho